d15331b25242c5a5d80118b139e7a27bb95076bb
apollo-core/src/main/java/com/ctrip/framework/foundation/internals/Utils.java
--- 
+++ 
@@ -1,33 +1,33 @@
 /*
- * Copyright 2021 Apollo Authors
+ * Copyright 2022 Apollo Authors
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
  * http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
  */
 package com.ctrip.framework.foundation.internals;
 
 import com.google.common.base.Strings;
 
 public class Utils {
   public static boolean isBlank(String str) {
     return Strings.nullToEmpty(str).trim().isEmpty();
   }
 
   public static boolean isOSWindows() {
     String osName = System.getProperty("os.name");
     if (Utils.isBlank(osName)) {
       return false;
     }
     return osName.startsWith("Windows");
   }
 }